speaker_verification/sql_utils.py

```diff
@@ -1,11 +1,22 @@
 import io
-from os.path import abspath, dirname, join, exists
+import logging
+import os
 import sqlite3
+from datetime import date
+from os.path import abspath, dirname, join, exists
 
 import numpy as np
 
+from speaker_verification.utils.logger import SpeakerVerificationLogger
 
 DATABASE_PATH = join(abspath(dirname(__file__)), "SQL", "sqlite.db")
+today = date.today()
+file_path = os.path.dirname(os.path.abspath(__file__))
+logger = SpeakerVerificationLogger(
+    name=__file__,
+    log_file=f'{file_path}/logs/logs-{today.strftime("%d-%m-%Y")}.log'
```

Comment:
I feel like this logfile string should be optional for `SpeakerVerificationLogger` with `f'{file_path}/logs/logs-{today.strftime("%d-%m-%Y")}.log'` being the default. That way we wouldn't have to import datetime, and os.path everywhere we want to log to the daily logfile.

Reply:
Good idea!

Reply:
Looks much cleaner now!